src/atoms/Spacer/spacer.scss

```diff
@@ -0,0 +1,31 @@
+ $base: .25;
+
+/** -- shared spacer util classes --
+  Common spacer sizing incrementing ru(.25) to ru(2.5)
+*/
+@for $i from 1 through 10 {
+  .top-#{$i} { margin-top: ru($base * $i); }
+  .bot-#{$i} { margin-bottom: ru($base * $i); }
+}
+
+.top-0 { margin-top: 0; }
+.bot-0 { margin-bottom: 0; }
+
+.top-11 { margin-top: ru(3); }
+.bot-11 { margin-bottom: ru(3); }
+
+.top-12 { margin-top: ru(6); }
+.bot-12 { margin-bottom: ru(6); }
+
+/* -- named spacer sizes --
+  Named spacer sizes uses @extend to save some
+  compiled space and provide you dev a map to the unit
```

Comment:
I kind of get why you're giving the dev the option to use "any" spacing option.. but it seems like giving them explicit permission to not use one of the 5. If you want to do this, perhaps consider giving the dev the option to simply use a number with the `space` prop in addition to the string `x-sm`, and then just inlining the conversion to `ru`? Also, it's not clear when / how these classes are used.